public/static/docs/api-reference/open.md

```diff
@@ -0,0 +1,188 @@
+# open()
+
+Opens a tracked file.
+
+```py
+dvc.api.open(path: str,
+             repo: str = None,
+             rev: str = None,
+             remote: str = None,
+             mode: str = "r",
+             encoding: str = None)
+```
+
+#### Usage:
+
+```py
+import dvc.api
+
+with dvc.api.open(
+        'get-started/data.xml',
+        repo='https://github.com/iterative/dataset-registry'
+        ) as fd:
+    # ... fd is a file descriptor that can be processed normally.
+```
+
+## Description
+
+Open file or model (`path`) tracked in a <abbr>DVC project</abbr> (by DVC or
```

Comment:
a bit too complicated, (`path`) .... (by DVC or Git) ... etc ... can we avoid these clarifications in ( ), instead make them part of the sentence?

Reply:
file or model sounds confusing too ... `data or model file` ?

Reply:
also, what about directories?

Reply:
Updated. open and read don't support directories last time I checked.